ui/components/ui/button/buttons.scss

```diff
@@ -282,12 +282,12 @@ input[type="submit"][disabled] {
 
     &--disabled,
     &[disabled] {
-      border-color: var(--color-warning-alternative);
+      border-color: var(--color-warning-default);
       color: var(--color-text-muted);
     }
 
     &:active {
-      border-color: var(--color-warning-alternative);
+      border-color: var(--color-warning-default);
     }
```

Comment:
### After


https://github.com/MetaMask/metamask-extension/assets/8112138/94b5d6af-ce90-4179-97a0-20a76a2cb255

Reply:
shouldn't the text be colored as well?

Reply:
Good question. This is from a deprecated button that I believe isn't used anywhere, so I'm just keeping it consistent with what is currently there. See Before screencast

Screenshot shows no usage of type warning for a button component 

![Screenshot 2024-06-05 at 3 10 48 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/4f8497be-2200-475e-8657-e5e2ab429f79)